app/models/cart_item.rb

```diff
@@ -0,0 +1,4 @@
+class CartItem < ApplicationRecord
+  belongs_to :cart
+  belongs_to :product
```

Comment:
You can also add relation of user and cartitem through cart
it will be something like this 
belongs_to :user , through: :cart